csvToJson.py

```diff
@@ -0,0 +1,152 @@
+import datetime
+
+import pandas as pd
+import json
+import os
+
+
+def convert_csv_to_json():
+
+    col = ["indicateur",
+           "period_date",
+           "valeur",
+           "mesure",
+           "short_indic",
+           "maille",
+           "indic_id",
+           "Code_Departement",
+           "Code_Region"]
+    df_propilot = pd.read_csv("exports/propilot.csv", usecols=col, sep=";")
+    # avoid null values
+    df_propilot = df_propilot.loc[~df_propilot.indicateur.isna()]
+    df_propilot['period_date'] = pd.to_datetime(df_propilot['period_date'])
+
+    file_name = 'france-relance-data-tableau-de-bord.txt'
+    clean_file(file_name)
+
+    # Récupération des indicateurs uniques
+    indicateurs = df_propilot.indicateur.unique()
+
+    for indicateur in indicateurs:
+        df_indicateur = df_propilot.loc[df_propilot.indicateur == indicateur]
+
+        data = {"code": df_indicateur["indic_id"].iloc[0],
+                "nom": df_indicateur["short_indic"].iloc[0],
+                "unite": df_indicateur["short_indic"].iloc[0]}
```

Comment:
```suggestion
        data = {"code": df_indicateur["indic_id"][0],
                "nom": df_indicateur["short_indic"][0],
                "unite": df_indicateur["short_indic"][0]}
```
Je serais partisant de préférer __getitem__ à iloc, ici, discutable

Reply:
plus clair à comprendre sans iloc